clang/lib/AST/ExprConstant.cpp

```diff
@@ -71,721 +71,704 @@
 
 using namespace clang;
 using llvm::APFixedPoint;
+using llvm::APFloat;
```

Comment:
DEFINITELY looks like you clang-formatted this entire file.  Please don't!  I can't see what the changes you made here were.